mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageMapperTest.java

```diff
@@ -127,23 +126,6 @@ void mailboxContainingMessagesShouldHaveTheGoodMessageCount() throws MailboxExce
         assertThat(messageMapper.countMessagesInMailbox(benwaInboxMailbox)).isEqualTo(5);
     }
 
-    @Test
-    void getMailboxCountersShouldReturnStoredValue() throws MailboxException {
```

Comment:
why did you removed this test?

Reply:
Because I removed the method it tested.

Short note: with reactor group retrieval is no longer needed.